apps/cowswap-frontend/src/common/updaters/orders/ExpiredOrdersUpdater.ts

```diff
@@ -35,14 +33,9 @@ export function ExpiredOrdersUpdater(): null {
 
         // Filter orders:
         // - Owned by the current connected account
-        // - Created in the last 5 min, no further
         // - Not yet refunded
-        const pending = expiredRef.current.filter(({ owner, creationTime: creationTimeString, refundHash }) => {
-          const creationTime = new Date(creationTimeString).getTime()
-
-          return (
-            owner.toLowerCase() === lowerCaseAccount && now - creationTime < EXPIRED_ORDERS_PENDING_TIME && !refundHash
-          )
+        const pending = expiredRef.current.filter(({ owner, refundHash }) => {
+          return owner.toLowerCase() === lowerCaseAccount && !refundHash
         })
 
         if (pending.length === 0) {
```

Comment:
not from this PR, but i would just remove the ELSE. the IF already returns early, so there's no point

Reply:
also delete the dead code

Reply:
It actually makes sense.
If we don't have any pending orders, we should not move further.
I see only one exit before, when `isUpdating.current` is truth, but it's different